tests/test_fieldset_sampling.py

```diff
@@ -126,8 +126,8 @@ def test_fieldset_sample(fieldset, xdim=120, ydim=80):
     lat = np.linspace(-80, 80, ydim, dtype=np.float32)
     v_s = np.array([fieldset.UV[0, 0., 70., x][1] for x in lon])
     u_s = np.array([fieldset.UV[0, 0., y, -45.][0] for y in lat])
-    assert np.allclose(v_s, lon, rtol=1e-7)
-    assert np.allclose(u_s, lat, rtol=1e-7)
+    assert np.allclose(v_s, lon, rtol=1e-4)
+    assert np.allclose(u_s, lat, rtol=1e-4)
```

Comment:
Make the tolerance here (and below) mode-dependent too? It's still strange that numpy-v2 leads to such bigger errors (but only on these few tests!)...